x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -472,3 +472,8 @@ export const RISKY_USERS_DOC_LINK =
 export const MAX_NUMBER_OF_NEW_TERMS_FIELDS = 3;
 
 export const BULK_ADD_TO_TIMELINE_LIMIT = 2000;
+
+export const DUPLICATE_OPTIONS = {
+  WITH_EXCEPTIONS: 'WITH_EXCEPTIONS',
+  WITHOUT_EXCEPTIONS: 'WITHOUT_EXCEPTIONS',
+};
```

Comment:
Now that we have subdomains, let's avoid using this file. These constants belong to `rule_management`.